.github/workflows/jams.yml

```diff
@@ -0,0 +1,30 @@
+name: JSON Annotated Music Specification for Reproducible MIR Research
+
+on: [push]
+#  push:
+#    branches: [ master ]
+#  pull_request:
+#    branches: [ master ]
```

Comment:
We should also have these enabled for PR

Reply:
should i remove the push one?

Reply:
We should have both, see https://github.com/librosa/librosa/blob/894942673d55aa2206df1296b6c4c50827c7f1d6/.github/workflows/ci.yml#L3-L9

Reply:
interesting, by adding both i can't really run the actions on my fork. i'm assuming that it's because i'm not in the main branch?

Reply:
Right; on your fork it's not a PR against master, and also not a push to master.  But once we merge this so the CI is live on the main repo, it should work again.

Reply:
understood, thanks :smile: